papers/Sepideh_Alassi_Bertram_Winter/PyCells_for_an_Open_Semiconductor_Industry

```diff
@@ -0,0 +1,270 @@
+:author: Sepideh Alassi
+:email: sepideh.alassi@iis.fraunhofer.de
+:institution: Fraunhofer IIS
+
+:author: Bertram Winter
+:email: Bertram.Winter@ams.com
+:institution: ams AG
+
+
+------------------------------------------------
+PyCells for an Open Semiconductor Industry
+------------------------------------------------
+
+.. class:: abstract
+
+	In modern semiconductor industry, automatic generation of parameterized and recurring layout structures plays an important role and should be present as a feature in Electronic Design Automation (EDA)-tools. Currently these layout generators are developed with a proprietary programming language and can be used with a specific EDA-tool. Therefore, the semiconductor companies find the development of the layout generators that can be used in all state of the art EDA-tools which support OpenAccess database appealing. The goal of this project is to develop computationally efficient layout generators with Python (PyCells), for ams AG technologies, that possess all the features of comprehensive layout generators.
```

Comment:
Typos in the abstract:

...In *the* modern semiconductor industry...

... proprietary programming language and can *only* be used ...

What is "ams"?